packages/table/src/V11Adapter/V11Adapter.types.ts

```diff
@@ -15,3 +15,13 @@ export type V11AdapterProps<T extends LGRowData> = PropsWithChildren<
       headerLabels?: { [key: string]: string };
     }
 >;
+
+export interface AdapterRowProps {
+  children?: React.ReactNode;
+  [key: string]: any;
+}
+
+export interface ValidDataType {
```

Comment:
Should `ValidDataType` extend, or be related to `LGRowData` in some way?

Reply:
LGRowData is what's consumed by `react-table` as Row data, and this is what the user is allowed to pass in as their own object, so it feels unrelated